### PR TITLE
Add cookbook to table: Multi-lingual Customer Support Response Pipeline

### PR DESCRIPTION
**Pull request by Ritesh-Udgata**
This pull request Adds another example to the cookbook table showcasing how we can use instill to make multi-lingual customer support response pipeline.

### CODE REVIEW

**Comment by Ritesh-Udgata** [comment]
please review this before mid night

**Comment by Ritesh-Udgata** [comment]
Kindly add hacktoberfest topic to your repo or add hacktoberfest accepted to this pull request.

File: examples/Multi_Lingual_Customer_Support_Response_Pipeline.ipynb [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "4eRArD3TnjOj"
+      },
+      "outputs": [],
+      "source": [
+        "from instill_sdk import InstillCoreClient\n",
```
Comment (GeorgeWilliamStrong) [inline examples/Multi_Lingual_Customer_Support_Response_Pipeline.ipynb:52]:
```suggestion
        "from instill_sdk import InstillCoreClient\n",
```
This is not how to use our SDK. Please refer to our [documentation](https://www.instill.tech/docs/sdk/python).

```Python
from instill.clients import init_core_client

client = init_core_client(api_token="instill_sk_***")
```

**Comment by GeorgeWilliamStrong** [comment]
Thank you for your PR! Please see the following feedback:

This notebook currently doesn't make use of any Instill AI services. If you want this to be accepted to our cookbook, please can you create a relevant VDP pipeline and update it to call that pipeline. Please also ensure that the formatting and style is consistent with the other examples in the cookbook repo.

**Review by GeorgeWilliamStrong — CHANGES_REQUESTED** [review]
Please address my comments

**Comment by Ritesh-Udgata** [comment]
Ok i will come up with some better idea in the future. For now i am closing this pull request.

**Pull request closed without merging**